Setting UI refactoring

Zerichen opened the pull request.

anishsarangi commented:
@Zerichen Please provide a proper commit message in every commit you make. 
Thank you !!

anishsarangi commented on css/settings.css:
```diff
@@ -1,58 +1,182 @@
+
+span.section {
+    text-align: center;
+    padding-left: 18px;
```
Instead of addressing padding of each, please use padding shorthand property with four values.
See this https://www.w3schools.com/css/css_padding.asp .
You could see this at a lot of places, please make the changes.
Thank you !

Zerichen replied:
Will fix that.

Zerichen commented:
@Zerichen Sorry those commits were in a rush and thanks for the link. 
I will describe the commit message a bit more here for people who might need to reference in the future:

initialize: Merged `settings.html` with `index.html`
experiment: Some basic setup for demonstrating the goal of this branch
polished version: a finished brand new setting UI interface within popup page
fix error: Updated the loading CSS file in `index.html` 
fix error: Fixed a problem with the style of the suggestion list in search box
refine tab button: Made tab has pressed effect for users to see which section there are in.
